data_rentgen/db/migrations/versions/2025-03-17_976168ee4f16_remove_partitions_from_dataset_names.py

```diff
@@ -0,0 +1,136 @@
+# SPDX-FileCopyrightText: 2024-2025 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+"""remove_partitions_from_dataset_names
+
+Revision ID: 976168ee4f16
+Revises: 15c0a22b8566
+Create Date: 2025-03-17 18:28:24.545234
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "976168ee4f16"
+down_revision = "15c0a22b8566"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # Create a Map old_id -> new_id
+    op.execute(sa.text("CREATE TEMP TABLE dataset_migration (id SERIAL PRIMARY KEY, old_id INT, new_id INT);"))
+    # Add new rows into dataset with 'short' name if such names doesn't exists
+    op.execute(
+        sa.text(
+            """
+            INSERT INTO dataset (name, location_id)
```

Comment:
`dataset` table also has `format` column

Reply:
actualy I'm not sure about this step cause all of this datasets already exists

Reply:
Dataset uniqueness is checked only for location_id + name